### PR TITLE
Add  react-markdown-editor-lite via single package.json w/ npm auto-u…

### PR DESCRIPTION
**Pull request by sylingd**
# Pull request template for new library

Pull request for issue: #13935
Related issue(s):

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

## Profile of the lib

* Git repository (required):https://github.com/HarryChen0506/react-markdown-editor-lite
* Official website (optional, not the repository):
* NPM package url (optional):https://www.npmjs.com/package/react-markdown-editor-lite
* License and its reference:[MIT](https://github.com/HarryChen0506/react-markdown-editor-lite/blob/master/LICENSE)
* GitHub / Bitbucket popularity (required):
  * Count of watchers:6
  * Count of stars:271
  * Count of forks:57
* NPM download stats (optional):
  * Downloads in the last day:206
  * Downloads in the last week:2448
  * Downloads in the last month:9654

## Essential checklist

* [x] I'm the author of this library
  * [x] I would like to add link to the page of this library on CDNJS on website / readme
* [x] This lib was not found on cdnjs repo
* [x] Doesn't already exist / has duplicate issue and PR
* [x] The lib has notable popularity
  * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [x] More than 800 downloads stats per month on npm registry
* [x] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [x] Has valid tags for each versions (for git auto-update)
* [x] Auto-update setup
* [x] Auto-update target/source is valid.
* [x] Auto-update filemap is correct.

## Git commit checklist

* [ ] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [x] The parent of the commit(s) in the PR is not older than 3 days.
* [ ] Pull request is sent from a non-master branch with a meaningful name.
* [x] Separate unrelated changes into different commits.
* [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [ ] Close corresponding issue in commit message
* [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/react-markdown-editor-lite/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "editor",
+    "parser",
+    "react",
+    "component",
+    "plugins",
+    "pluggable"
+  ],
+  "author": "HarryChen && ShuangYa",
+  "license": "MIT",
+  "homepage": "https://github.com/HarryChen0506/react-markdown-editor-lite#readme",
```
Comment (sashberd) [inline ajax/libs/react-markdown-editor-lite/package.json:16]: I think this is the home page https://harrychen0506.github.io/react-markdown-editor-lite/... Am I wrong?

Comment (sylingd) [reply ajax/libs/react-markdown-editor-lite/package.json:16]: > I think this is the home page https://harrychen0506.github.io/react-markdown-editor-lite/... Am I wrong?

Yes, but in face, this website is not maintained frequently, and the content is relatively old. Only GitHub has complete content

Comment (sylingd) [reply ajax/libs/react-markdown-editor-lite/package.json:16]: Yes, but in fact, this website is not maintained frequently, and the content is relatively old. Only GitHub has complete content. So I think it ’s better to use GitHub as its homepage

Comment (sylingd) [reply ajax/libs/react-markdown-editor-lite/package.json:16]: > I think this is the home page https://harrychen0506.github.io/react-markdown-editor-lite/... Am I wrong?

Yes, but in fact, this website is not maintained frequently, and the content is relatively old. Only GitHub has complete content. So I think it ’s better to use GitHub as its homepage

Comment (sashberd) [reply ajax/libs/react-markdown-editor-lite/package.json:16]: OK. LGTM

**Comment by xtuc** [comment]
I moved it to the new bot https://github.com/cdnjs/packages/pull/10/files

**Pull request closed without merging**